packages/blade/src/components/Popover/_decisions/decisions.md

```diff
@@ -0,0 +1,275 @@
+# Popover API Decisions <!-- omit in toc -->
+
+The popover component is used to provide context as well as enable users to take certain actions on it. They are designed to display relevant information without requiring the user to navigate to a different page or dialog. Popovers are triggered on click or onload.
+
+<img src="./assets/popover-thumbnail.png" width="380" alt="Popover Thumbnail" />
+
+- [Design](#design)
+- [Basic Features](#basic-features)
+- [`Popover` API](#popover-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+    - [Controlled](#controlled)
+    - [Uncontrolled](#uncontrolled)
+    - [On initial load](#on-initial-load)
+    - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Future Scope](#future-scope)
+- [Tooltips vs Popovers vs Guided Popovers](#tooltips-vs-popovers-vs-guided-popovers)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Question](#open-question)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=2875-40870&mode=design&t=mTbpUCQOr7kHk6UT-0) to all variants of the Popover component
+
+## Basic Features
+
```

Comment:
mention somehwere that popovers are not auto dismissible unline tooltip

Reply:
I think this could be a good point for comparison.

Reply:
Added